tests/apm_tracing_e2e/test_single_span.py

```diff
@@ -45,7 +47,9 @@ def test_parent_span_is_single_span(self):
         _assert_single_span_event(spans[0], "parent.span.single_span_submitted", is_root=True)
 
     def setup_child_span_is_single_span(self):
-        self.req = weblog.get("/e2e_single_span?parentName=parent.span&childName=child.span.single_span_submitted")
+        self.req = weblog.get(
```

Comment:
It is the same name as setup_parent_span_is_single_span, it's better to. use another name, like `self.req_child`

Reply:
Why is that? Isn't there isolation on the `self` object between the tests?

Reply:
yes it is. Just a way to be a little bit cleaner. But you're right, consider it just as a suggestion :)

Reply:
I had that at my first attempt but it seemed redundant, and it was easier to copy paste and modify the code between the tests :)